tests/api/administrator.contact.1.0.0.Cest.php

```diff
@@ -32,8 +32,8 @@ public function WebserviceIsAvailable(ApiTester $I)
 		// Checking for 204 code since there are no contacts before inserting them
 		$I->seeResponseCodeIs(204);
 		$I->seeResponseIsJson();
-		$I->seeHttpHeader('Webservice-name', 'contact');
-		$I->seeHttpHeader('Webservice-version', '1.0.0');
+		$I->seeHttpHeader('X-Webservice-name', 'contact');
```

Comment:
@javigomez I have changed the custom header names to X-something because that is the agreed standard of the custom headers. So you can change them in the tests